plugins/index.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: plugin
+title: Plugins
+---
+# 1. Plugins
+
+Plugins are tools to extend and modify the functionality of Foreman. The core of Foreman is designed to be lean, to maximize flexibility and minimize code bloat. Plugins offer custom functions and features so that each user can tailor their environment to their specific needs. Foreman plugins are implemented as [rail engines](http://guides.rubyonrails.org/engines.html) that are packaged as [gems](http://guides.rubygems.org/what-is-a-gem/) and thus easily installed into Foreman.
+
+## 1.1 Available Plugins
```

Comment:
Popular plugins maybe? It gives me the impression these are the only plugins available, it might be me though.